Capstone_Project/frontEnd/src/components/Home/Homepage.css

```diff
@@ -0,0 +1,211 @@
+.totalPage
+{
+    display: flex;
+}
+/************/
+.nav{
+    width:80px;
+    height: 1100px;
+    background-color: rgb(237, 229, 229);
+    border-right:1px solid black;
+    position: fixed;
+ }
+ .logo
+ {
+     margin-top: 20px;
+     width: 80px;
+     height: 80px;
+ }
+ .livebook
+ {
+    width: 80px;
+    height: 80px;
+    margin-top: 10px;
+ }
+ .profile
+ {
+    width: 80px;
+    height: 80px;
+    margin-top: 350px;
+ }
+
+ /**************/
+
+ .container
+ {
+    width: 100%;
+ }
+
+ /*****************/
+ .introduction
+ {
+    border: 1px solid black;
+    margin-left: 80px;
+    height: 200px; 
+    padding: 5px;
+    display: flex;
+    background-color: rgb(76, 76, 196);
+ }
+ .liveHeros
+ {
+   margin-left: 300px;
+ }
+ .textmove
+ {
+   margin-top: 20px;
+   margin-left: 20px;
+ }
+.maths
+{
+   font-size: 20px;
+   color: rgb(220, 213, 213);
+}
+.mouse
+{
+   display: block;
+   margin-left: 675px;
+   margin-right: auto;
+   margin-top: 200px;
+}
+.mouse1
+{
+   display: block;
+   margin-left: 700px;
+   margin-right: auto;
+   /* margin-top: 200px; */
+}
+.courseinfo
+{
+   margin-left: 100px;
+}
+.course
+{
+   text-align: center;
+   display: flex;
+   margin-left: 100px; 
+   font-size: 25px;
+}
+.course:hover
+{
+   color: red;
+}
+.leftborder
+{
+   border: 1px solid black;
+   width: 350px;
+}
+.container2
+{
+   width: 100%;
+}
+.container2flex
+{
+   display: flex;
+}
+.units
+{
+    display: flex;
+    width: 1200px;
+    border: 2px solid black;
+    margin-left: 30px;
+}
+.a
+{
+    width: 25px;
+    height: 25px;
+    margin-top: 10px;
+    margin-left: 20px;
+    padding: 3px;
+}
+.maths
+{
+    color:  rgb(239, 235, 236);
+    font-size: 20px;
+    
+}
+.b
+{
+    width: 10px;
+    height: 10px;
+    margin-top: 10px;
+    text-align: center;
+    margin-left: 20px;
+    padding: 5px;
+    border: 1px solid black;
+}
+.b:hover
+{
+    color:  rgb(210, 19, 25);
+}
+.c
+{
+    font-weight: 500;
+    width: 600px;
+    margin-top: 10px;
+    margin-left: 30px;
+    padding: 5px;
+}
+.c:hover
+{
+    color:rgb(17, 16, 17);
+}
+.dropdown
+{
+    position: absolute;
+    right: 50px;
+}
+.unitshow
+{
+   margin-top: 20px;
+}
+.subunits
+{
+    display: flex;
+    margin: 5px;
+}
+.content
+{
+    margin-left: 100px;
+    font-weight: 1;
+}
+.circle
+{
+    width: 20px;
+    height: 20px;
+    border-radius: 100px;
+    border: 1px solid black;
+    margin-right: 10px;
+    margin-top: 10px;
+}
+.c
+{
+    font-weight: 500;
+    width: 600px;
+    /* min-height: 10px;
```

Comment:
PLease remove unwanted code and use selectors and combinators

Reply:
Resolved